src/main/flight/mixer.c

```diff
@@ -314,10 +307,28 @@ static void applyFlipOverAfterCrashModeToMotors(void)
             }
         }
 
+        const float flipRateLimit = mixerConfig()->crashflip_rate * 10.0f; // eg 35 = no power by 350 deg/s
+        float flipRateAttenuator = 1.0f;
+        if (flipRateLimit > 0) {
+            // get the gyro rate for the fastest controlled axis
+            float gyroRate = 0.0f;
+            if (signYaw != 0) {
+                // only yaw is used
+                gyroRate = fabsf(gyro.gyroADCf[FD_YAW]);
```

Comment:
What is this supposed to do? Yaw is not used in turtle mode. You cannot flip the quad using yaw. Or am I missing something?

Reply:
Yaw is quite helpful for getting out of trees. Haven't checked to see, but if this is part of the new smarter turtle mode stuff just for flipping over then yaw doesn't really make sense I would agree.

Reply:
Thanks, that makes sense

Reply:
I checked yaw today.  If full yaw stick is applied, full motor power goes to diagonally opposite motors, and keeps being applied until the tilt angle changes.  If the quad is inverted on grass, and one of the two diagonal motors is in relatively free air, that motor spins up and flips the quad, while the other is stalled.
To be honest I'm not certain how tilt angle would change with a yaw movement, I think that with a pure yaw movement, tilt angle won't change, and you'll have full power until either you centre the sticks, or the quad leaps out of it's position and the tilt angle changes a lot.
In trees the new code may or may not be as useful as before.  I think it will be similar.  Each time the turtle mode is active, up to full power is applied; power only reduces if the quad moves significantly.

Reply:
So inverted yawing still works? Then I made a mistake by thinking about the angle attenuation the wrong way (as an absolute value instead of a difference)

Reply:
Yes, if inverted, the yaw stick just spins diagonally opposing motors.  

On a smooth floor it would yaw spin and try to lift off at the same time :-). Not much use at all.  

But if you're stuck in some situation where you need to rotate the quad, primarily, it will do that.  When inverted in grass, one of the two motors tends to become stuck, and won't spin, the other motor will then we get a diagonal flip.

These behaviours are what we had before, and are not changed by this PR.